topdown/eval.go

```diff
@@ -2735,7 +2735,9 @@ func (e evalVirtualComplete) eval(iter unifyIterator) error {
 		return nil
 	}
 
-	if len(e.ir.Rules) > 0 && len(e.ir.Rules[0].Head.Args) > 0 {
+	// When evaluating the full extent, skip functions.
+	if len(e.ir.Rules) > 0 && len(e.ir.Rules[0].Head.Args) > 0 ||
+		e.ir.Default != nil && len(e.ir.Default.Head.Args) > 0 {
```

Comment:
That's one long `if` that would look so much better in Rego :D

Reply:
Yeah... I guess there might also be a more concise way here, since we know that `ir.Empty()` is _false_, which is `len(e.ir.Rules()) == 0 && e.ir.Default == nil` 🤔